src/extension.ts

```diff
@@ -49,13 +52,21 @@ async function init(context: vscode.ExtensionContext, git: ApiImpl, repository:
 	context.subscriptions.push(vscode.window.registerUriHandler(uriHandler));
 	context.subscriptions.push(new FileTypeDecorationProvider());
 
-	const prManager = new PullRequestManager(repository, telemetry, git);
+	const credentialStore = new CredentialStore(telemetry);
```

Comment:
I've extracted the credential store instantiation here, so we can reuse it in initializing `GithubRemoteSourceProvider`.